### ISSUE TITLE
Install on Ubuntu 20.04, image artifacts in slide pane

### ISSUE THREAD

**Issue by imgifty** [qupath/qupath#628]
## Bug report

**Describe the bug**

**To Reproduce**
1. Download release 0.2.3 on  Ubuntu 20.04
2. Extract archive
3. Run qupath from bin directory
4. Open project and import mirax slide

**Expected behavior**
Slide without weird artifacts

**Screenshots**
![Screenshot from 2020-10-25 15-29-30](https://user-images.githubusercontent.com/6561496/97109997-e96d5100-16d6-11eb-9c68-d20863276593.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - QuPath Version 0.2.3 (also doesn't work on 0.2.1)

**Comment by petebankhead** [comment]
Hi @imgifty, have you checked if it could be the same as https://github.com/qupath/qupath/issues/355?
This is referenced in the docs [here](https://qupath.readthedocs.io/en/latest/docs/intro/installation.html#linux).

See also
* https://github.com/qupath/qupath/issues/627
* https://github.com/libvips/libvips/issues/1862#issuecomment-715237292
* https://forum.image.sc/t/change-image-provider-after-import/42502/3
* https://forum.image.sc/t/install-and-start-qupath-ubuntu-20-04/39580/11

In summary, the specific problem with some .mrxs images isn't something that can be resolved directly inside QuPath - rather, it depends upon OpenSlide and/or its dependencies. You might be able to fix things by installing/updating OpenSlide via a package manager.

(There *is* a more general QuPath issue inasmuch as it needs a better way to support OpenSlide in the future... I'll create another issue to discuss that.)

**Comment by imgifty** [comment]
Hello @petebankhead ,
I reinstalled Pixman-0.40.0 and this indeed fixed my problem, thanks a lot!

Thanks for the speedy help!

**Comment by phromo** [comment]
@imgifty can you help out by providing how you reinstalled pixman? The latest version I can find is 0.38.4-0ubuntu1 for focal.

**Comment by phromo** [comment]
I rebuilt the deb myself. If anyone else goes looking, you might find my packages for ubuntu 20.04-focal here: https://www.lupin.se/~phromo/pixman/ -- use with caution, I'm a noob at debian package building.

**Comment by imgifty** [comment]
> @imgifty can you help out by providing how you reinstalled pixman? The latest version I can find is 0.38.4-0ubuntu1 for focal.

Hello phromo, I built it using following guide: http://www.linuxfromscratch.org/blfs/view/svn/general/pixman.html .
Sorry for the late reply.

**Comment by phromo** [comment]
Thanks! The packages I provided should be easier to work with, e.g. to uninstall/upgrade once the ubuntu repositories provides 0.40. Manually replacing libraries can cause a mess when you want to upgrade to the next ubuntu version.

**Comment by petebankhead** [comment]
Quick update, I started suffering this problem too with Ubuntu 20.04 LTS.

As a workaround, I create a `launch_qupath.bsh` script inside the QuPath directory with the following contents:

```
#!/bin/bash

DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
echo $DIR
export LD_PRELOAD=$DIR/lib/app/libpixman-1.so.0
$DIR/bin/QuPath-0.2.3
```

**Comment by jdvylder** [comment]
> I rebuilt the deb myself. If anyone else goes looking, you might find my packages for ubuntu 20.04-focal here: https://www.phromo.se/~phromo/pixman/ -- use with caution, I'm a noob at debian package building.

Hi Phromo

this link seems to be broken. Is there another location where we can find your build?

thanks

**Comment by phromo** [comment]
I'll have a look at restoring the link. There's currently a power outage due to grid work, expected to have it working again in roughly 2 hours after writing this post.

**Comment by jdvylder** [comment]
Phromo, thanks for the fast reply and fix

**Comment by RSchmitzHH** [comment]
Had the same issue with QuPath 0.3.2, also on Ubuntu 20.04. 

Even though this may not be the preferred/ primary solution: Ubuntu 22.04 has a newer version of pixman with which the artifacts do not occur. QuPath 0.3.2 runs smoothly with it.
